Examples/Swift/OfflinePackExample.swift

```diff
@@ -42,13 +44,10 @@ class OfflinePackExample_Swift: UIViewController, MGLMapViewDelegate {
         }
     }
 
-    deinit {
-        // Remove offline pack observers.
-        print("Removing offline pack notification observers")
-        NotificationCenter.default.removeObserver(self)
```

Comment:
It’s still important to clean up observations when deinitializing (or when the view controller is about to disappear, depending). The application may allow the user to navigate away from the offline pack–downloading view controller before the download completes. If that happens, the view controller shouldn’t leak memory due to a lingering observation.